docs/azmcp-commands.md

```diff
@@ -266,6 +266,29 @@ azmcp extension az --command "storage account show --name <account-name> --resou
 azmcp extension az --command "vm list --resource-group <resource-group>"
 ```
 
+
+### Azure Load Testing Operations
+```bash
+# Execute load test command to get all the commands details
+azmcp loadtesting loadtest
```

Comment:
Also add entries to /e2eTests which should be the prompts that you expect to trigger the tool to be called by an Agent